### ISSUE TITLE
Training loss does not decrease, training acc is equal to 0

### ISSUE THREAD

**Issue by Jia-Xueyu** [ronghuaiyang/arcface-pytorch#99]
Hello, I use the casia-webface to train the model. However, the train loss loss does not drop and remains at around 14. When I use the pre-train model Resnet18-110, the loss can decrease. Can you elaborate on the process of training data？Thank you.

**Comment by disguise6** [comment]
After setting s = 6, m = 2, loss decreased and acc increased to 1. But I use my own dataset, not casia-webface.

**Comment by abcde-bit** [comment]
> Hello, I use the casia-webface to train the model. However, the train loss loss does not drop and remains at around 14. When I use the pre-train model Resnet18-110, the loss can decrease. Can you elaborate on the process of training data？Thank you.

Hi brother. I also have the problem. How did you solve the problem at last.
![image](https://user-images.githubusercontent.com/59106543/226818426-cf52238f-5a41-4895-8503-84a6f1a8dfd7.png)
